docs/website/sidebars.js

```diff
@@ -23,6 +23,7 @@ const sidebars = {
       label: 'About Ouroboros',
       items: [
         'about-ouroboros/index',
+        'about-ouroboros/utxo-hd',
```

Comment:
Aren't the reports too low-level for this section? I would expect these to be linked from the "For Developers" section.

Reply:
Also, these are only references. The contents of the `.md` file shouldn't be too technical, but I think it's fine to link documents for further reading, so I don't think they have to move

Reply:
> The main report is also listed in `about-ouroboros/References`. Should we move that one too then?

I think that would make sense.

Reply:
In other words my main objection is that it feels strange to have a top level (sidebar) link for a very specific feature.